src/__test__/components/data-exploration/differential-expression-tool/DiffExprManager-regression-load-on-second-render.test.jsx

```diff
@@ -44,47 +39,135 @@ const mockStore = configureMockStore([thunk]);
 
 const experimentId = '1234';
 
-const storeState = {
-  cellSets: {
-    ...cellSetsInitialState,
-    hierarchy: [],
-    properties: {},
+const backendStatus = {
+  [experimentId]: {
+    status: {
+      pipeline: {
+        startDate: '2021-01-01T00:00:00',
+      },
+    },
   },
+};
+
+const storeState = {
   genes: {
-    ...genesInitialState,
-    focused: undefined,
+    selected: [],
+  },
+  cellInfo: {
+    focus: {
+      store: 'genes',
+      key: 'C',
+    },
+  },
+  cellSets: {
```

Comment:
can you include the mock cellSets we have and use it here?

Reply:
Addressed. There were 2 mock cell sets files: `cellSets.mock` and `mockCellSets`. I chose to use `cellSets.mock` because the mocks are closer to real conditions. I've updated the remaining tests to use that mock too and deleted the `mockCellSets`.